src/filters/dasher.c

```diff
@@ -10775,6 +10784,10 @@ static const GF_FilterArgs DasherArgs[] =
 		"- br: use LL-HLS with byte-range for segment parts, pointing to full segment (DASH-LL compatible)\n"
 		"- sf: use separate files for segment parts (post-fixed .1, .2 etc.)\n"
 		"- brsf: generate two sets of manifest, one for byte-range and one for files (`_IF` added before extension of manifest)", GF_PROP_UINT, "off", "off|br|sf|brsf", GF_FS_ARG_HINT_EXPERT},
+	{ OFFS(dashll), "DASH Low Latency type\n"
+		" - cte: use Chunked Transfer Encoding for segments\n"
+		" - sf: use separate files for segment parts (post-fixed .1, .2 etc.)\n",
```

Comment:
I think we should use the word "Segment Sequence" somewhere to make it searchable for users